### DISCUSSION TITLE
How to report a new ROM?

### DISCUSSION THREAD

**Discussion by malespiaut** [mamedev/discussions#104] [Q&A]
Hello,

I've just bought an expansion cartridge for the French version of the VTech Learning Window (see https://github.com/mamedev/mame/blob/master/src/mame/skeleton/learnwin.cpp).

The chip is labelled “SPR128A-086”, and is named “mon ordinateur junior - Cartouche-Programme - La Gymnastique du Cerveau”.

Note: “mon ordinateur junior” is the name of “Learning Window” for the French market.

I would like to know how to properly report my finding.

I have taken photos that'll post in the discussion later today or during the week.

Also: I do **not** have any equiment, nor knowledge, about how to dump this cart. If anyone is ready to help me with that, it'll be with pleasure.

Best regards.

**Comment by startaq** [comment]
Team Europe can probably help you. You can contact them here: https://team-europe.blogspot.com
